openmetadata-spec/pom.xml

```diff
@@ -51,6 +51,12 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
+            <version>2.1.6</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Where are we using this?

Reply:
I was getting compile error on IDE. Will remove it and debug why I was seeing that issue.